src/libs/PaymentUtils.ts

```diff
@@ -0,0 +1,88 @@
+import {SvgProps} from 'react-native-svg';
+import {FC} from 'react';
+// eslint-disable-next-line import/no-named-default
+import {default as BankAccountModel} from './models/BankAccount';
+import getBankIcon from '../components/Icon/BankIcons';
+import CONST from '../CONST';
+import * as Localize from './Localize';
+import Fund from '../types/onyx/Fund';
+import BankAccount from '../types/onyx/BankAccount';
+
+type AccountType = 'debitCard' | 'bankAccount';
+
+type PaymentMethod = {
+    description: string;
+    icon: FC<SvgProps>;
+    iconSize: number;
+} & BankAccount &
+    Fund;
+
+/**
+ * Check to see if user has either a debit card or personal bank account added
+ */
+function hasExpensifyPaymentMethod(fundList: Record<string, Fund>, bankAccountList: Record<string, BankAccount>): boolean {
+    const validBankAccount = Object.values(bankAccountList).some((bankAccountJSON) => {
+        const bankAccount = new BankAccountModel(bankAccountJSON);
+        return bankAccount.isDefaultCredit();
+    });
+
+    // Hide any billing cards that are not P2P debit cards for now because you cannot make them your default method, or delete them
+    const validDebitCard = Object.values(fundList).some((card) => card?.accountData?.additionalData?.isP2PDebitCard ?? false);
+
+    return validBankAccount || validDebitCard;
+}
+
+function getPaymentMethodDescription(accountType: AccountType, account: BankAccount['accountData'] & Fund['accountData']): string {
+    if (account) {
+        if (accountType === CONST.PAYMENT_METHODS.BANK_ACCOUNT && account?.accountNumber) {
+            return `${Localize.translateLocal('paymentMethodList.accountLastFour')} ${account?.accountNumber?.slice(-4)}`;
+        }
+        if (accountType === CONST.PAYMENT_METHODS.DEBIT_CARD) {
+            return `${Localize.translateLocal('paymentMethodList.cardLastFour')} ${account?.cardNumber?.slice(-4)}`;
+        }
+    }
+    return '';
+}
```

Comment:
```suggestion
type AccountType = BankAccount['accountType'] | Fund['accountType'];

function getPaymentMethodDescription(accountType: AccountType, account: BankAccount['accountData'] | Fund['accountData']): string {
    if (account) {
        if (accountType === CONST.PAYMENT_METHODS.BANK_ACCOUNT && 'accountNumber' in account) {
            return `${Localize.translateLocal('paymentMethodList.accountLastFour')} ${account.accountNumber?.slice(-4)}`;
        }
        if (accountType === CONST.PAYMENT_METHODS.DEBIT_CARD && 'cardNumber' in account) {
            return `${Localize.translateLocal('paymentMethodList.cardLastFour')} ${account.cardNumber?.slice(-4)}`;
        }
    }
    return '';
}
```

Reply:
I'm not sure about 
```
type AccountType = BankAccount['accountType'] | Fund['accountType'];
```
since both are returning `string | type` type

Reply:
Maybe lets change it to just `string` or just to `type AccountType = BankAccount['accountType'];`